components/AssetPage/AssetPage.tsx

```diff
@@ -88,6 +89,13 @@ const AssetPage = ({ assetID, data, files, renders }) => {
       setPreviewImage(defaultImg)
     }
 
+    // Set preview image size
+    fetch(`https://cdn.polyhaven.com/asset_img/thumbs/${assetID}.png?format=png`).then(
+      res => res.blob()
+    ).then((res) => {
+      setPreviewImageSize(res.size)
+    })
+
```

Comment:
I came up with this hacky way of getting the size of the preview images in the browser. I'd much rather fetch this data from the db, but I thought I should present this idea too.

Somewhat related, I noticed something strange:  the size of the blob is the same as the size of the image if I download it manually (with `wget`), but the displayed size of the zip file is less than the size of the downloaded zip (`8.2MB` vs `7.78MB`). A similar difference is present when I view the same file on the website (https://polyhaven.com/a/wood_cabinet_worn_long) too: `6.69MB` vs `7.00MB`. Maybe we don't add the size of a file to the sum that we display?